src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -55,6 +58,12 @@ class ResourceTemplateForm extends Component {
     return buttons
   }
 
+  mandatorySuperscript = (propMandatory) => {
+    if (JSON.parse(propMandatory)) {
+      return <sup><FontAwesomeIcon className="asterick text-danger" icon={faAsterisk} /></sup>
```

Comment:
Why is this different than the child input components? Why not also use `RequiredSuperscript ` here? Is there a nuance I am not seeing?